webapp/components/searchable_user_list.jsx

```diff
@@ -156,10 +156,7 @@ export default class SearchableUserList extends React.Component {
         }
 
         return (
-            <div
-                className='filtered-user-list'
-                style={this.props.style}
```

Comment:
Why are you removing this prop? If some bad style is being passed in, we should remove it where it's being passed in, not by making the prop non-functional